internal/service/elbv2/target_group_test.go

```diff
@@ -5558,16 +5688,17 @@ resource "aws_vpc" "test" {
 }`, rName)
 }
 
-func testAccTargetGroupConfig_albUpdateSlowStart(rName string, slowStartDuration int) string {
+func testAccTargetGroupConfig_albUpdateSlowStart(rName string, slowStartDuration int, loadBalanceAlgorithmType string) string {
```

Comment:
Added to check for invalid slow start configuration.  `weighted_random` does not support slow start.